homeassistant/components/go2rtc/__init__.py

```diff
@@ -62,11 +62,17 @@
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up WebRTC."""
     url: str | None = None
-    if not (url := config[DOMAIN].get(CONF_URL)):
+    if not (configured_by_user := DOMAIN in config) or not (
+        url := config[DOMAIN].get(CONF_URL)
+    ):
         if not is_docker_env():
+            if not configured_by_user:
+                return True
             _LOGGER.warning("Go2rtc URL required in non-docker installs")
             return False
         if not (binary := await _get_binary(hass)):
+            if not configured_by_user:
+                return True
```

Comment:
We may want to fail here, we expect `go2rtc` to be available in our docker images

Reply:
Yes